cloudify_azure/auth/oauth2.py

```diff
@@ -29,10 +31,12 @@
 
 import requests
 import jwt
+import warnings
 
 # Used to implement connection retrying
 from requests.packages import urllib3
 from msrestazure.azure_active_directory import AADMixin
+from msrestazure.azure_cloud import (AZURE_PUBLIC_CLOUD, AZURE_CHINA_CLOUD)
```

Comment:
Are we pinned to a particular version?

Reply:
Meaning is this part of 4.0.0?

Reply:
Not sure If I got your point here for these two comments, which version ?